### ISSUE TITLE
Sticky elements on small screens/zoom block content

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#849]
## Describe the bug
Accessibility audit issue 106: At 320 px viewport width, the sticky header, toggle context button, and minimap button block page content, making it difficult for the user to navigate and read content, blocking some footer content entirely,  and obscuring the visible focus on some elements.

## To reproduce
1. Go to an /objects or /collections page
2. Set browser width to 320px (or 1028px at 400% zoom)
3. See issue

## Error message or screenshots

<img width="458" height="268" alt="Image" src="https://github.com/user-attachments/assets/ab3a7a4d-5b06-4001-8251-a153626b950a" />

## Impact on your work
This is a violation of 2.4.11 Focus Not Obscured (Minimum), and blocks access for users who rely on screen magnification.

**Comment by helrond** [comment]
closed by #859
